examples/pytorch_simple.py

```diff
@@ -0,0 +1,168 @@
+"""
+Optuna example that optimizes multi-layer perceptrons using PyTorch.
+
+In this example, we optimize the validation accuracy of hand-written digit recognition using
+PyTorch and MNIST. We optimize the neural network architecture as well as the optimizer
+configuration. As it is too time consuming to use the whole MNIST dataset, we here use a small
+subset of it.
+
+We have the following two ways to execute this example:
+
+(1) Execute this code directly.
+    $ python pytorch_simple.py
+
+
+(2) Execute through CLI.
+    $ STUDY_NAME=`optuna create-study --storage sqlite:///example.db`
+    $ optuna study optimize mxnet_simple.py objective --n-trials=100 --study $STUDY_NAME \
+      --storage sqlite:///example.db
+
+"""
+
+from __future__ import print_function
+
+import tempfile
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+import torch.optim as optim
+import torch.utils.data
+from torchvision import datasets
+from torchvision import transforms
+
+
+if torch.cuda.is_available():
+    DEVICE = torch.device('cuda')
+else:
+    DEVICE = torch.device('cpu')
+BATCHSIZE = 128
+CLASSES = 10
+DIR = tempfile.mkdtemp()
+EPOCHS = 10
+LOG_INTERVAL = 10
+
+
+class Net(nn.Module):
+    def __init__(self, trial):
+        super(Net, self).__init__()
+        self.layers = []
+        self.dropouts = []
+
+        n_layers = trial.suggest_int('n_layers', 1, 3)
+        dropout = trial.suggest_uniform('dropout', 0.2, 0.5)
+        input_dim = 28*28
+        for i in range(n_layers):
+            output_dim = int(trial.suggest_loguniform('n_units_l{}'.format(i), 4, 128))
+            self.layers.append(nn.Linear(input_dim, output_dim))
+            self.dropouts.append(nn.Dropout(dropout))
+            input_dim = output_dim
+
+        self.layers.append(nn.Linear(input_dim, CLASSES))
+
+        for idx, layer in enumerate(self.layers):
+            setattr(self, 'fc{}'.format(idx), layer)
+
+        for idx, dropout in enumerate(self.dropouts):
+            setattr(self, 'drop{}'.format(idx), dropout)
+
+    def forward(self, data):
+        data = data.view(-1, 28*28)
+        for layer, dropout in zip(self.layers, self.dropouts):
+            data = F.relu(layer(data))
+            data = dropout(data)
+        return F.log_softmax(self.layers[-1](data), dim=1)
+
+
+def get_mnist():
+    # Load MNIST dataset
```

Comment:
Please add trailing period here (and to the other comment lines).

Reply:
Formatted all comments.